Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: Every Order, Resolution, or Vote to which the Concurrence of the Senate and House of Representatives may be necessary (except on a question of Adjournment) shall be presented to the President of the United States; and before the Same shall take Effect, shall be approved by him, or being disapproved by him, shall be repassed by two thirds of the Senate and House of Representatives, according to the Rules and Limitations prescribed in the Case of a Bill.
+3: Every other Bill which has been presented to the House of Representatives and the Senate, and has been approved and signed by all Representatives and Senators from several of the States, shall be sent to the Chief Executive of each of those States. If he should approve the Bill, it shall be considered approved by his State; if not, he shall send it to the Legislature of his State. If, after consideration, the Legislature of the State approves and signs the Bill in keeping with that State's Constitution, it shall be considered approved by the State. When at least two States have approved the Bill, it shall become law in those states which have so approved it, for a period not to exceed six years from first approval.
 
 Section 8
-1: The Congress shall have Power To lay and collect Taxes, Duties, Imposts and Excises, to pay the Debts and provide for the common Defence and general Welfare of the United States; but all Duties, Imposts and Excises shall be uniform throughout the United States;
+The Congress shall have Power:
 
-2: To borrow Money on the credit of the United States;
+1:  To lay and collect Taxes, Duties, Imposts, and Excises, on commerce among the several States and with foreign nations; to pay the Debts and provide for the common Defense of these United States; but all Duties, Imposts and Excises shall be uniform throughout these United States; and no Tax, Duty, Impost, or Excise shall apply to transactions occurring wholly within one State, nor shall any resident of any State be compelled by law to participate in a transaction other than the payment of such Taxes, Duties, Imposts, and Excises.
 
-3: To regulate Commerce with foreign Nations, and among the several States, and with the Indian Tribes;
+2: To borrow Money on the credit of these United States, when such credit shall be approved and a payment plan funded by laws passed as described in Article I, Section 7, Part 2;
 
-4: To establish an uniform Rule of Naturalization, and uniform Laws on the subject of Bankruptcies throughout the United States;
+3: To regulate Commerce with foreign Nations, and that which crosses borders among the several States;
```

Comment:
Here we come to a significant change in response to one of the worst Supreme Court decisions in history: Wickard v. Filburn.

That Court decision, decided entirely by Justices appointed by Franklin D Roosevelt, upheld charges against Mr. Filburn, who was charged for violating wheat price and production controls, because he grew wheat entirely within his own lands in the state of Ohio, in order to feed to his farm animals, and the amount he grew was in excess of Federal regulatory limits.

The Supreme Court found that, because Mr. Filburn's activity producing wheat for himself caused him not to need to buy wheat at the market, it had an impact on interstate commerce. This decision essentially gave Congress authority over all economic activity in these United States, an absurd expansion of their authority, and the basis for many fascistic government actions in the 78 years since.